Content.Shared/Cargo/CargoBountyHistoryData.cs

```diff
@@ -0,0 +1,46 @@
+using Robust.Shared.Serialization;
+using Content.Shared.Cargo.Prototypes;
+using Robust.Shared.Prototypes;
```

Comment:
You gotta sort your imports
```suggestion
using Content.Shared.Cargo.Prototypes;
using Robust.Shared.Prototypes;
using Robust.Shared.Serialization;
```